features/gm.ts

```diff
@@ -97,7 +98,28 @@ export class GroupMessageController implements GroupMessageGetter, GroupMessageL
             ],
             encryptKey: groupCtx.publicKey,
         });
```

Comment:
1. lamport time is not used here.
2. Lamport time is not sufficient enough to produce a good sort if any devices go offline and generate events at the same time.
3. We should consider vector clock or other clocks in the future.